activities/documentation/move-merge-delete-pages.md

```diff
@@ -22,9 +22,9 @@ A redirect to can only redirect to one other new page.
 Note: this means a file needs to be kept in the old location.
 
 A `redirect_from` is an instruction in the new page.
-This works a bit like updating everyone's address book with your new address.
+This works a bit like mail arriving at the right building, and a friendly neighbour bringing it to your new door.
```

Comment:
US instead of UK English.
```suggestion
This works a bit like mail arriving at the right building, and a friendly neighbor bringing it to your new door.
```